Content.Client/UserInterface/Systems/Ghost/Controls/Roles/GhostRoleButtonsBox.xaml.cs

```diff
@@ -34,7 +31,7 @@ public GhostRolesEntry(string name, string description, bool hasAccess, Formatte
                 {
                     button.RequestButton.Disabled = true;
 
-                    if (reason != null && !reason.IsEmpty)
+                    if (reason is { IsEmpty: false })
```

Comment:
Please don't make random style changes like this in the future.

Reply:
Okay